src/components/autocomplete/js/autocompleteController.js

```diff
@@ -236,7 +243,7 @@ function MdAutocompleteCtrl ($scope, $element, $mdUtil, $mdConstant, $mdTheming,
    */
   function onListLeave () {
     noBlur = false;
-    if (!hasFocus) ctrl.hidden = true;
+    ctrl.hidden = shouldHide();
```

Comment:
The `shouldHide()` method now takes into account the `hasFocus` variable, so yes, this should work exactly as before.